### DISCUSSION TITLE
Secret key from UUID?

### DISCUSSION THREAD

**Discussion by singularity-s0** [OpenTreeHole/backend#20] [General]
<img width="254" alt="image" src="https://user-images.githubusercontent.com/12184989/136356447-da775ae8-fb98-405d-96e0-c976877b7298.png">
Why default to this? This could raise security concerns, especially when the attacker has shell access.

**Comment by hasbai** [comment]
这只是供开发测试时用的默认配置，部署上线时应用自己的 key 覆盖默认配置
